src/store/vis-ui-config-slice.ts

```diff
@@ -35,9 +33,9 @@ export const initialState: VisUiConfigState = {
     options: DEFAULT_OPTIONS,
     outputType: 'EVENT',
     layout: {
-        columns: [],
-        filters: [],
-        rows: [],
+        columns: EMPTY_STRING_ARRAY,
+        filters: EMPTY_STRING_ARRAY,
+        rows: EMPTY_STRING_ARRAY,
```

Comment:
Why is this preferable over `[]`?

Reply:
Changed back. We agreed is more readable and there isn't any reason to use the constants since this is an initial value.